package.json

```diff
@@ -352,7 +352,8 @@
       },
       {
         "command": "confluent.copyResourceUri",
-        "title": "Copy URI",
+        "title": "Copy URL",
+        "comment": "Using the label 'URL' here instead of 'URI' since these are web URLs, not URIs, and 'URL' matches the terminology used in scaffolding templates.",
```

Comment:
> Using the label 'URL' here instead of 'URI' since these are web URLs, not URIs

nit: Aren't web URLs also URIs?

Reply:
Yes, all URLs are URIs, but URL is both the more common and more specific term. Academically, URIs were an ill-gotten concept from the get-go in the 90s (canonical example being ... book ISBNs), spawning confusion the world over. "An identifier combined with a location and mechanism for access is a URL. If it's just a uniform spelling of an identifier ONLY, then is a URI."

But the motivation here is pedestrian --- just get the darn labels to line up between the CTA and the scaffolding project fields so that the poor users will have a higher degree of certainty that they're about to copy the right thing to then paste into the form.